Remove AgentSubordinationRelation

tfrancart opened the pull request:
Fix #22 
Removes AgentSubordinationRelation - and removes "isSuperiorOf" and "isInferiorOf" in keywords detected in the XML.
To be discussed

florenceclavaud commented:
The old, now deleted, AgentSubordinationRelation was generated from an eaccpfRelation of type associative, as we cannot for now use a cpf hierarchical relation to connect two persons in our system. So in the v2, we should have AgentControlRelation (which replaces this old relation) declared in both ASSOCIATIVE_RELATION_CONFIG and HIERARCHICAL_RELATION_CONFIG (where it already existed). We should also probably make a lot of changes in the keywords file, to have arcrole values that correspond to the shortcuts in RiC-0 0.2 (but this is for the future of our system in fact...)

tfrancart commented:
@florenceclavaud  I have added AgentControlRelation in associative relations, and modified test cases accordingly. Please have a look and let me know.

florenceclavaud reviewed (approved):
Everything is OK there. Approved
